src/client/datakit_client_9p.ml

```diff
@@ -577,6 +577,8 @@ module Make(P9p : Protocol_9p_client.S) = struct
         (fun hash -> node_of_hash t hash >>*= fn)
 
     let fast_forward t commit =
+      let path = Datakit_path.of_steps_exn (branch_dir t) in
+      FS.make_dirs t.fs ~base:[] path >>*= fun () ->
```

Comment:
It was failing if the branch didn't exist. Are you sure we are doing the right test in `create`?